tests/writeResult/writeresult-getdeletedcount-002.phpt

```diff
@@ -19,12 +19,13 @@ $bulk->delete(['x' => 1]);
 
 $result = $manager->executeBulkWrite(NS, $bulk, new MongoDB\Driver\WriteConcern(0));
 
-var_dump($result->getDeletedCount());
+throws(function() use ($result) {
+    $result->getDeletedCount();
+}, MongoDB\Driver\Exception\LogicException::class, 'getDeletedCount');
```

Comment:
Even if it's optional, adding an assertion to the throwing method costs nothing.